jaeger-core/src/main/java/io/jaegertracing/internal/JaegerObjectFactory.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2018, Uber Technologies, Inc
```

Comment:
nit: The Jaeger Authors

Reply:
Good point. This happens to be copied from other files, but we should correct them.